### ISSUE TITLE
Tag relases?

### ISSUE THREAD

**Issue by BenMorel** [pop-os/gnome-shell-theme#92]
Hi, would it be possible to tag releases for this project, and ideally keep them in sync with the `gtk-theme` releases?

I'd like to maintain RPMs of your amazing themes for other distros to use, but having to deal with `master` is not ideal.

Thank you!

**Comment by jackpot51** [comment]
We do prefer to tag releases for all Pop!_OS projects. @isantop do you have any thoughts on this?

**Comment by isantop** [comment]
@BenMorel For our themes, the Debian packages are always built from `master`, so that is the best way to go. The shell theme specifically doesn't really iterate through releases, and instead everything just goes directly through master.

**Comment by mmstick** [comment]
This really applies to all of our projects -- commits reaching `master` for projects which have been released are already being actively tested by QA, and represent changes that are being pushed out to users today, rather than withholding patches until they can be wrapped up into a single tag release.

Although it's still useful to have tags from time to time to mark milestones, and for Linux distributions that aren't willing to adopt patches in a rolling release manner.

**Comment by BenMorel** [comment]
Makes sense, I guess; I really didn't expect that `master` would be QA'd and production ready at all times, this is quite different from what I'm used to.

I don't see any other development branch though, does that mean that you're maintaining another centralized repo internally, and only have your commits pushed to GitHub by QA when ready?

Also, what makes the [gtk-theme](https://github.com/pop-os/gtk-theme/releases) different, as this one does have releases?

**Comment by mmstick** [comment]
We don't have a specific branch for development. Every non-master branch is a development branch that only exists until the point that the pull request for that branch has been reviewed, tested by QA, and approved.

Once rebased and merged, our CI proceeds to build the latest commit on master into a debian package, and publishes it to our Proposed PPA. Shortly thereafter, we publish all the packages in proposed into stable. We simply use it as a buffer for holding new packages until we can release them all at the same time, and testing out changes that affect our ISOs.

**Comment by BenMorel** [comment]
@mmstick Thanks for the explanation! What about `gtk-theme`, why does this one need a special treatment?

**Comment by mmstick** [comment]
There's no reason for special treatment. We can also tag releases here.
